bldsys/cmake/template/gradle/app.build.gradle.in

```diff
@@ -65,11 +69,14 @@ android {
 
 	buildFeatures {
 		viewBinding true
+		prefab true
 	}
 }
 
 dependencies {
-	implementation 'androidx.appcompat:appcompat:1.3.0'
-	implementation 'com.google.android.material:material:1.4.0'
-	implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-}
\ No newline at end of file
+	implementation 'androidx.appcompat:appcompat:1.5.1'
+	implementation 'com.google.android.material:material:1.7.0'
+	implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+	implementation 'androidx.core:core:1.9.0'
+	implementation 'androidx.games:games-activity:1.2.2-alpha01'
```

Comment:
is there a non-alpha version we could use?

Reply:
1.2.1 is the formal release. The static library release started with 1.2.2-alpha01, highly recommend this one. Release cycle is pretty fast: it will beta next week, then formal release.

Reply:
thank you!